lib/robots/dor_repo/gis_delivery/load_geoserver.rb

```diff
@@ -35,19 +35,15 @@ def perform_work
 
           logger.debug "Workspace: #{workspace_name} ready"
 
-          if layer['vector'] && layer['vector']['format'] == 'PostGIS'
-            create_vector(connection, layer['vector'], workspace_name)
-          elsif layer['raster'] && layer['raster']['format'] == 'GeoTIFF'
-            create_raster(connection, layer['raster'], workspace_name)
+          if vector? && layertype == 'PostGIS'
+            create_vector(connection, workspace_name)
+          elsif raster? && layertype == 'GeoTIFF'
+            create_raster(connection, workspace_name)
           else
-            raise "load-geoserver: #{bare_druid} has unknown layer format: #{layer}"
+            raise "load-geoserver: #{bare_druid} has unknown layer format: #{layertype}"
```

Comment:
I don't think it's an issue but the layer format in the exception previously would have included more than just the format name?